### PR TITLE
Todo list project PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

I created a todo list project with webpack bundle. Please check my code and let me know your thoughts.

Thanks.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="input-container">
+          <input
+            type="text"
+            name="todo"
+            id="todo"
+            class="input-field"
+            placeholder="Add new item..."
+          />
+          <button class="add-btn">
+            <i class="fa-solid fa-arrow-turn-down fa-rotate-90"></i>
```
Comment (lenkon) [inline src/index.html:28]: Hi @mertkantaroglu, It's better to keep the inline styles in a CSS file for HTML and CSS best practices.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
+body {
+  background-color: rgb(240, 240, 240);
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  height: 100vh;
+}
+
+.title {
+  display: flex;
```
Comment (lenkon) [inline src/style.css:10]: Hi @mertkantaroglu, the display property could be defined once and then reuse, thus following the HTML & CSS best practices.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  if (input.value !== '') {
+    addToDo(toDoItems(), input.value);
+    input.value = '';
+    setToDo(toDoItems());
+    renderToDoList(toDoItems());
+  }
+});
+
+// Edit Event Listener
+toDoList.addEventListener('click', (e) => {
```
Comment (lenkon) [inline src/index.js:37]: Hi @mertkantaroglu, it is better to give meaningful names for variables according to JavaScript clean code best practices, eg: e could be an event, etc. Please change it wherever necessary.

**Review by lenkon — COMMENTED** [review]
Hi @mertkantaroglu ,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Aspects to improve ♻️

- [x] Check the comments under the review.

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (mertkantaroglu) [reply src/index.html:28]: Hi Lenkon,
In here I used Fontawesome icon. In order to implement the icon I needed to use this tag in HTML.

File: src/style.css (hunk 2 above)
Comment (mertkantaroglu) [reply src/style.css:10]: Thanks for the information. I'll fix it accordingly.

File: src/index.js (hunk 3 above)
Comment (mertkantaroglu) [reply src/index.js:37]: Thanks, Lenkon. I changed them according to your suggestion.

**Pull request merged**